Source/Reporters/DataTableReporter.jsl

```diff
@@ -0,0 +1,140 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+
+Names Default To Here( 0 );
+
+/*
+	Class: UtDataTableReporter
+		---Prototype---
+		class UtDataTableReporter inherits UtReporter
+		---------------
+		Saves results from tests as rows in a JMP table
+*/
+Define Class(
+	"UtDataTableReporter",
+	Base Class( "UtReporter" ),
+	
+	//Table to store results
+	dt = .;
+	
+	_init_ = Method({}, 
+		this:dt = New Table( "Test Results",
+			visibility("private"),
+			Add Rows( 0 ),
+			New Script(
+				"Summary",
+				Tabulate(
+					Show Control Panel( 0 ),
+					Add Table( Row Table( Grouping Columns( :result ) ) )
+				)
+			),
+			New Column( "DateTime",        Numeric,    "Continuous", Format( "m/d/y h:m:s", 23, 0 ),
+				Input Format( "m/d/y h:m:s", 0 ), Set Values([]) ),
+			New Column( "label",           Expression, "None",    Set Values({}) ),
+			New Column( "Case Name",       Character,  "Nominal", Formula( Eval List( :label )[1] ) ),
+			New Column( "Test Name",       Character,  "Nominal", Formula( Eval List( :label )[2] ) ),
+			New Column( "Assertion Label", Character,  "Nominal", Formula( Eval List( :label )[3] ) ),
+			New Column( "Assertion ID",    Numeric,    "Nominal", Formula( Eval List( :label )[4] ) ),
+			New Column( "test expr",       Expression, "None",    Set Values({}) ),
+			New Column( "description",     Character,  "Nominal", Set Values({}) ),
+			New Column( "lre",             Expression, "None",    Set Values({}) ),
```

Comment:
I think `lre` should be `Numeric` and `"Continuous"`. And all the `Empty()` below for `lre` can become `.`.

Reply:
@emccorkle, I don't believe numeric works due to potential tests like this:
``` JSL
ut assert value( [2 2], [1 2] );
```

Reply:
You are right. Sorry about that.